tests/test_package_generation.py

```diff
@@ -8,6 +8,7 @@
 import typing
 
 import pytest
+import pytest_venv
```

Comment:
What about this one-liner instead?
```suggestion
import pytest_venv  # type: ignore[import-not-found]
```